crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -367,11 +367,11 @@ pub struct ShardedGossipLocalState {
     round_map: RoundStateMap,
     /// Metrics that track remote node states and help guide
     /// the next node to gossip with.
-    metrics: Metrics,
```

Comment:
wonder if Metrics should be pulled up a layer now, since this struct is wrapped in a mutex.

Reply:
Yeh there's no reason for metrics to be sync when ShardedGossipState is not shared

Reply:
Or is this because you are reading it somewhere else?

Reply:
Yes, we need to be able to update the metrics from remote calls in addition to gossip calls.